libqpdf/QPDF.cc

```diff
@@ -1055,12 +1047,11 @@ QPDF::read_xrefStream(qpdf_offset_t xref_offset)
 {
     bool found = false;
     if (!this->m->ignore_xref_streams) {
-        int xobj;
-        int xgen;
+        QPDFObjGen x_og;
         QPDFObjectHandle xref_obj;
         try {
             xref_obj = readObjectAtOffset(
-                false, xref_offset, "xref stream", -1, 0, xobj, xgen);
+                false, xref_offset, "xref stream", QPDFObjGen(-1), x_og);
```

Comment:
This will need to be `QPDFObjGen(0, 0)` after my upcoming commit.

Reply:
I used QPDFObjGen() throughout for the default case. Is that ok?

Reply:
Yes, I have no objection to it for any other use than readObjectAtOffset because of the explicit use of 0 to mean something.

Reply:
Added explicit 0,0's for those calls.